test/test_models.py

```diff
@@ -31,21 +35,49 @@ def test_mfcc(self):
         assert out.size() == (batch_size, num_classes, 2)
 
 
-class TestMelResNet:
+class TestMelResNet(common_utils.TorchaudioTestCase):
 
     def test_waveform(self):
-
         batch_size = 2
-        num_features = 200
-        input_dims = 100
-        output_dims = 128
-        res_blocks = 10
-        hidden_dims = 128
-        pad = 2
+        n_time = 200
+        n_freq = 100
+        n_output = 256
+        n_res_block = 10
+        n_hidden = 128
+        kernel_size = 5
 
-        model = _MelResNet(res_blocks, input_dims, hidden_dims, output_dims, pad)
+        model = _MelResNet(n_res_block, n_freq, n_hidden, n_output, kernel_size)
```

Comment:
nit: these changes should be in a separate PR

Reply:
Thanks. I will rebase it on #751 .